examples/projections/cyl/cyl_oblique_mercator.py

```diff
@@ -0,0 +1,81 @@
+r"""
+Oblique Mercator projection
+===========================
+
+Oblique configurations of the cylinder give rise to the oblique Mercator projection.
+It is particularly useful when mapping regions of large lateral extent in an oblique
+direction. Both parallels and meridians are complex curves. The projection was
+developed in the early 1900s by several workers.
+
+The projection is set with **o** or **O**. There are three different specification
+ways (**a**\|\ **A**, **b**\|\ **B**, **c**\|\ **C**) available. For all three
+definitions, the upper case letter mean the projection pole is set in the southern
+hemisphere [Default is northern hemisphere]. Align the y-axis with the optional
+modifier **+v**. The figure size is set with *scale* or *width*.
+
+
+1. Using the origin and azimuth
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**oa**\|\ **oA**\ *lon0/lat0/azimuth/scale*\[**+v**] or
+**Oa**\|\ **OA**\ *lon0/lat0/azimuth/width*\[**+v**]
+
+The central meridian is set by *lon0/lat0*.
+The oblique equator is set by *azimuth*.
+
+
+2. Using two points
+~~~~~~~~~~~~~~~~~~~
+
+**ob**\|\ **oB**\ *lon0/lat0/lon1/lat1/scale*\ [**+v**] or
+**Ob**\|\ **OB**\ *lon0/lat0/lon1/lat1/width*\ [**+v**]
+
+The central meridian is set by *lon0/lat0*.
+The oblique equator is set by *lon1/lat1*.
+
+
+3. Using the origin and projection pole
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**oc**\|\ **oC**\ *lon0/lat0/lonp/latp/scale*\ [**+v**] or
+**Oc**\|\ **OC**\ *lon0/lat0/lonp/latp/width*\ [**+v**]
+
+The central meridian is set by *lon0/lat0*.
+The projection pole is set by *lonp/latp*.
+"""
+
+# %%
```

Comment:
I feel it's a little weird to show three mini maps in one gallery thumbnail image (https://pygmt-dev--3451.org.readthedocs.build/en/3451/projections/index.html#cylindric-projections). Since it's just three different ways to specify one projection, maybe we can show three separate maps in this example and only show one of them in the gallery summary page?

Reply:
I combined the three maps into one figure, because the maps for the options [**Oa**](https://www.pygmt.org/dev/projections/cyl/cyl_oblique_mercator_1.html#sphx-glr-projections-cyl-cyl-oblique-mercator-1-py) and [**Ob**](https://www.pygmt.org/dev/projections/cyl/cyl_oblique_mercator_2.html#sphx-glr-projections-cyl-cyl-oblique-mercator-2-py) are quite high. And having all three maps in the current size below each other makes the website quite long. But your are right, the thumbnail image looks not optimal.
In commit https://github.com/GenericMappingTools/pygmt/pull/3451/commits/cd15b7dafaf1dfacc4f7448da9cc82f9be16f97c I tested showing separate maps for using a much smaler sizes for the maps. For the thumbnail image I picked the map for **Oc**, as the ascept ratio seems to fit best to the thumbnail image card.

Reply:
> And having all three maps in the current size below each other makes the website quite long.

I prefer to have the figure below each subsection, which is easier to read IMHO. Let's hear what others think about it.

Reply:
Ah, I have missed this point from your first review comment here. I just worked on removing the subplot. If having separate code blocks within each subsection, I think it's OK to show the maps separatly, even though the maps have a larger height. However, having them all tree below each other at the end looked not good.

Reply:
I like the new layout.